bin/pfioh

```diff
@@ -21,11 +21,42 @@ import  pudb
 # and related dependencies.
 sys.path.insert(1, os.path.join(os.path.dirname(__file__), '../pfioh'))
 try:
-    import  pfioh
+    import pfioh
+    import mount_dir
+    import swift_store
 except:
     sys.path.insert(1, os.path.join(get_python_lib(True, True), 'site-packages'))
     sys.path.insert(1, os.path.join(get_python_lib(True, True), 'site-packages/pfioh'))
-    import  pfioh
+    sys.path.insert(1, os.path.join(get_python_lib(True, True), 'site-packages/mount_dir'))
```

Comment:
Yes, I have separated them into two different files. Also in #200 and #202, I am passing the two classes from these two packages. This I have added so as to improve the previous implementation where I was instantiating the classes in pfioh.py with if else statements.